wazo_dird/plugins/default_json/http.py

```diff
@@ -77,6 +78,57 @@ def get(self, profile):
         return response
 
 
+class LookupByUUID(LegacyAuthResource, DisplayAwareResource):
+    def __init__(
+        self, lookup_service, favorite_service, display_service, profile_service, auth_client
+    ):
+        self.lookup_service = lookup_service
+        self.favorite_service = favorite_service
+        self.display_service = display_service
+        self.profile_service = profile_service
+        self.auth_client = auth_client
+
+    @required_acl('dird.directories.lookup.{profile}.{xivo_user_uuid}.read')
```

Comment:
remove the `user_uuid` prefix to avoid adding a xivo in our public API

Reply:
Do I do the same in the swagger doc?

Reply:
nevermind, we did not create another key without the prefix in the token data :(

Reply:
Ah that's why I tried and it kept failing on me!